layer5-ng/src/sections/Blog-single/index.js

```diff
@@ -38,73 +42,74 @@ const BlogSingle = () => {
+                    {/*  <img src={BlogThumb} alt="prime blog page" />*/}
+                    {/*  <div className="post-meta">*/}
+                    {/*    <span>*/}
+                    {/*      <em> 15 </em> SEP*/}
+                    {/*    </span>*/}
+                    {/*  </div>*/}
+                    {/*</div>*/}
+                    {/*<div className="post-content-block">*/}
+                    {/*  <p className="entry-one">*/}
+                    {/*    Coracoes costumes grandeza com observou horrivel mas.*/}
+                    {/*    Amor tive fara de dado esse em. Ve es couberam oh*/}
+                    {/*    garrafal mulheres. Dourados duzentos voz lustroso diz*/}
+                    {/*    discutir ahi. Luzindo no do tremula na so fallava.*/}
+                    {/*    Evitava tropheu curiosa.*/}
+                    {/*  </p>*/}
+                    {/*  <p className="entry-two">*/}
+                    {/*    Coracoes costumes grandeza com observou horrivel mas.*/}
+                    {/*    Amor tive fara de dado esse em. Ve es couberam oh*/}
+                    {/*    garrafal mulheres. Dourados duzentos voz lustroso diz*/}
+                    {/*    discutir ahi. Luzindo no do tremula na so fallava.*/}
+                    {/*    Evitava tropheu curiosa ou agitado os acceite si*/}
+                    {/*    assiste. Voz veio veja tez digo cres.*/}
+                    {/*  </p>*/}
+                    {/*  <blockquote>*/}
+                    {/*    You never change things by fighting the existing*/}
+                    {/*    reality.To change something, build a new model.*/}
+                    {/*  </blockquote>*/}
+                    {/*  <p className="entry-three">*/}
+                    {/*    Ao corrida ar queriam reparae do imposta acoitar do.*/}
+                    {/*    Qualidades intimativa aferventar ira acompanhar mau*/}
+                    {/*    capitulada enfraquece. Em apavorar ficarmos cantante se*/}
+                    {/*    ia blasonou eu comprido.*/}
+                    {/*  </p>*/}
 
-                      <p className="entry-four">
-                        Coracoes costumes grandeza com observou horrivel mas.
-                        Amor tive fara de dado esse em. Ve es couberam oh
-                        garrafal mulheres. Dourados duzentos voz lustroso diz
-                        discutir ahi. Luzindo no do tremula na so fallava.
-                        Evitava tropheu curiosa ou agitado os acceite si
-                        assiste. Voz veio veja tez digo cres.
-                      </p>
-                    </div>
-                    <div className="post-info-block">
-                      <div className="tags">
-                        <span>Tags:</span>
-                        <Link to="#">App</Link>
-                        <Link to="#">Landing</Link>
-                        <Link to="#">SASS</Link>

```

Comment:
@dhruv0000 understanding that the tags in the blogs are used as predefined categories for blog posts, how would we go about marrying up Strapi’s data model to these?